samples/peripheral/802154_phy_test/Kconfig

```diff
@@ -0,0 +1,80 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "Kconfig.zephyr"
+
+menu "PHY Test Tool configuration"
+
+choice
+	prompt "Device Mode"
+	default BOTH
+	help
+	  Select the device mode
+
+config DUT_MODE
```

Comment:
IMO it'd be good to add some common prefix to the app-related configs (something like `PTT_`), it'd be easier to identify and search for them in menuconfig.

Reply:
I'm not sure how to differentiate app-related configs from non-app-related ones. All of the configs in Kconfig are app-related in my opinion, should all of them carry a PTT_ prefix?

Reply:
Yes, it's quite obvious all of them should be app-specific since they're all in the app-specific Kconfig file - all of them should carry the prefix.